src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java

```diff
@@ -101,15 +98,11 @@ protected void initChannel(SocketChannel channel) throws Exception {
         cp.addLast("httpClientCodec", new HttpClientCodec());
 //        cp.addLast("defaltor", new HttpContentDecompressor());
         cp.addLast("chunkAggregator", new HttpObjectAggregator(maxFrameSize));
-        cp.addLast("idleTimeoutHandler", new IdleStateHandler(
-            20000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT,
-            TimeUnit.MILLISECONDS));
-        cp.addLast("idleDisconnectHandler", new IdleDisconnectHandler(
-            2000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT));
+        cp.addLast("idleDisconnectHandler", new XioIdleDisconnectHandler(
+              20000,
```

Comment:
I only see that having been done for [XioServerTransport](https://github.com/xjdr/xio/blob/master/src/main/java/com/xjeffrose/xio/server/XioServerTransport.java#L91).  This PR is for doing a similar change in HttpClientConnector.

Should XioClientConfig be modified to add a client idle timeout?